apps/www/pages/index.js

```diff
@@ -39,20 +40,26 @@ const MarketingPage = () => {
 
   return (
     <Frame raw meta={meta} isOnMarketingPage={true} hasOverviewNav>
-      <Marketing />
+      <Marketing data={data} />
     </Frame>
   )
 }
 
 export default MarketingPage
 
 export const getStaticProps = createGetStaticProps(async (client) => {
-  const data = await client.query({
+  const apiData = await client.query({
     query: MARKETING_PAGE_QUERY,
   })
+  const datoData = await getCMSClient().query({
+    query: MarketingLandingPageCmsDocument,
+  })
   return {
     props: {
-      data,
+      data: {
```

Comment:
Any particular reason, why these object are merged?

Reply:
ease of use. not necessary tho